javascript/javascript1/week2/main.js

```diff
@@ -26,3 +26,60 @@ function getFullname(
 
 const fullname1 = getFullname("Aaditri", "Sarkar", true, true);
 const fullname2 = getFullname(" ");
+
+//Event application
+const days = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+let todayNumber;
+function today(day) {
+  todayNumber = days.indexOf(day) + 1;
+}
+
+let eventday;
+function eventWeekday(id) {
+  switch (id) {
+    case 1:
+      eventday = "Sunday";
+      break;
+    case 2:
+      eventday = "Monday";
+      break;
+    case 3:
+      eventday = "Tuesday";
+      break;
+    case 4:
+      eventday = "Wednesday";
+      break;
+    case 5:
+      eventday = "Thursday";
+      break;
+    case 6:
+      eventday = "Friday";
+      break;
+    case 7:
+      eventday = "Saturday";
+  }
+  return eventday;
+}
+
+let eventDayWouldBe;
+function getEventWeekday(eventdayNumber) {
+  let dayNumber = eventdayNumber + todayNumber;
+  if (dayNumber > 6) {
+    eventDayWouldBe = dayNumber % 7;
+    console.log("Event day will be " + eventWeekday(eventDayWouldBe));
+  } else {
+    eventDayWouldBe = dayNumber;
+    console.log("Event day will be " + eventWeekday(eventDayWouldBe));
+  }
+}
+
+today("Sunday");
+getEventWeekday(8);
```

Comment:
Really great work. Cool to see you implement a switch statement 👌

OBS! 😊
Have you tested for different input values? What happens if you test getEventWeekday(6)? And why might that be? 

In the assignment description, there is a little hint to investigate new Date in js to generate the value of the current day. 

Lastly, consider how you could make one assignment for eventDayWouldBe that works for all eventDayNumber inputs, using the remainder operator. 💡

You're almost there. 🙌  Please let me know, if you want to talk it more thoroughly through 😊

Reply:
Thanks again. I have tried to modify it according to the requirement. If you can check and review.